tools/ansible.rst

```diff
@@ -52,3 +52,20 @@ To push for a change, start by cloning the repository::
 Then modify and push to the same repository. If you are in the group `admins`,
 then you will be able to push.  If not, then you can send the patch on
 gluster-infra mailling list.
+
+Fetching a PR from github
+-------------------------
+
+If anyone open a PR on github, you can merge it (after proper review) using the
+following process, for the PR number 4. I will assume that the repository is setup
+like this with 2 remote, 1 to fetch and 1 to push::
+
+    git add remote bastion ssh://salt-master.gluster.org/srv/git_repos/public
```

Comment:
I could maybe merge that with the previous section, and use a trick with push url.